internal/controller/controllers/error_utils.go

```diff
@@ -30,3 +34,12 @@ func isClientError(err error) bool {
 		return false
 	}
 }
+
+func imageBeingCreated(err error) bool {
```

Comment:
Maybe it should be more generic, like `IsHTTPError(id)`?

Reply:
Done